docs/src/modules/components/DemoSandbox.js

```diff
@@ -148,6 +148,7 @@ function getTheme(outerTheme) {
   if (outerTheme.spacing) {
     resultTheme.spacing = outerTheme.spacing;
   }
+  resultTheme.applyDarkStyles = outerTheme.applyDarkStyles;
```

Comment:
This is suspicious. `resultTheme` is supposed to return the default theme of Material UI & Joy UI (depending on the case), but `applyDarkStyles` is a custom helper of MUI's branding theme. 

```suggestion
```

Reply:
> So it seems that we can't have it here, it will break once people copy & paste the demos.

It won't break the copy & paste of the demos but I don't this line is needed, so thanks for calling it out.